File: Food.Data/Models/Bag.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Food.Data.Models
{
    public class Bag
    {
        public long Id { get; set; }
        public long UserId { get; set; }
        public string Name { get; set; }
        public bool UpdateData { get; set; }
        public string ConstituentsJSON { get; set; }
        public long NutritionId { get; set; }
        public Nutrition Nutrition { get; set; }
    }
}

File: Angular/food-app/src/app/models/bag.ts
import { BagItemNode } from './BagItemNode';
import { Food } from './food';
import { FoodItemNode } from './FoodItemNode';
import { Nutrition } from './Nutrition'

export class Bag {
  constructor(
    public id: number,
    public name: string,
    public updateData: boolean = false,
    public nutrition: Nutrition = new Nutrition(0,0,0,0,0),
    public foods: FoodItemNode[] = new Array<FoodItemNode>(),
    public bags: BagItemNode[] = new Array<BagItemNode>()) {

  }
}
